bigbluebutton-client/branding/default/style/css/BBBDefault.css

```diff
@@ -306,7 +306,7 @@ DataGrid {
 
 .presentationUploadButtonStyle, .pollStartButtonStyle, .presentationBackButtonStyle, .presentationBackButtonDisabledStyle, .presentationForwardButtonStyle, .presentationForwardButtonDisabledStyle,
 .presentationFitToWidthButtonStyle, .presentationFitToPageButtonStyle, .presentationDownloadButtonStyle,
-.sharedNotesSaveButtonStyle, .sharedNotesNewButtonStyle, .sharedNotesFormatButtonStyle
+.saveButtonStyle, .sharedNotesNewButtonStyle, .sharedNotesFormatButtonStyle, .copyButtonStyle

```

Comment:
Same thing for the sharedNotesSaveButtonStyle - just create a new style called chatSaveButtonStyle and leave the other styles.